### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by samreid** [phetsims/gravity-force-lab-basics#23]
I'm looking into PhET-iO instrumentation as part of #22 and wondering if there has been a code review yet, or if one has been scheduled.  Note this code review should include new common code that the simulation uses.

**Comment by jessegreenberg** [comment]
A code review has not happened yet. This sim is using code from inverse-square-law-common, which should also be code reviewed. There is still some active work going on for inverse-square-law-common, for instance 
https://github.com/phetsims/inverse-square-law-common/issues/23 and  https://github.com/phetsims/inverse-square-law-common/issues/21. Once those are complete, it will be time for dev testing and a code review of coulombs-law, gravity-force-lab-basics and inverse-square-law-common.

Does the code review need to happen before  #22 can begin? If so should we prioritize GFL:B over coulombs-law? We have been assuming that  coulombs-law would go out before GFL:B, but review and testing might need to come first now. @ariel-phet @samreid does that sounds correct?

**Comment by samreid** [comment]
@kathy-phet commented on the schedule with respect to code review in . https://github.com/phetsims/gravity-force-lab-basics/issues/22#issuecomment-355676563

**Comment by jessegreenberg** [comment]
Thanks! Does a code review need to happen before #22 can proceed?

**Comment by samreid** [comment]
No, we can begin on #22 and start leveraging PhET-iO instrumentation before the code review.

**Comment by mbarlow12** [comment]
@samreid do you mind if I close this issue and remake a new one for Code Review?

**Comment by samreid** [comment]
@mbarlow12 a new issue sounds great, close at your convenience.
